CMakeLists.txt

```diff
@@ -235,6 +238,13 @@ install(DIRECTORY "${NVFUSER_SRCS_DIR}/"
       DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}/nvfuser"
       FILES_MATCHING PATTERN "*.h" )
 
+add_custom_target(build_flatbuffer_config
```

Comment:
Did we forget to include `FlatBuffers.cmake` in the standalone build?

Reply:
Yes, since we use the PyTorch third party FlatBuffers.

Reply:
This error happens when including `FlatBuffers.cmake` with standalone build:
```
CMake Error at /home/rspring/workspace/nvfuser/third_party/flatbuffers/CMakeLists.txt:380 (add_library):
  add_library cannot create target "flatbuffers" because another target with
  the same name already exists.  The existing target is an interface library
  created in source directory "/home/rspring/workspace/release_pytorch".  See
  documentation for policy CMP0002 for more details.


CMake Error at /home/rspring/workspace/nvfuser/third_party/flatbuffers/CMakeLists.txt:384 (target_compile_options):
  target_compile_options may only set INTERFACE properties on INTERFACE
  targets
```